### ISSUE TITLE
Clean up old variables in color.scss once branding standardization work is done

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#2664]
This is a follow-up ticket for https://github.com/mozilla/foundation.mozilla.org/pull/2637

The branding standardization work is currently still WIP so we are keeping some to-be-removed variables in `colors.scss` for now to make the ongoing clean up work easier. Once the work is done we should remove old variables from `colors.scss`.
